gridded_etl_tools/dataset_manager.py

```diff
@@ -453,7 +455,7 @@ def run_etl(
 
         """
         # Find the dataset class (e.g. CHIRPSPrelim05) from its name string
-        dataset_class = self.get_dataset_manager_from_name(dataset_name)
+        dataset_class = self.get_subclass(dataset_name)
```

Comment:
This can be for a later PR, but now that we're getting around to cleaning up this area of the class, I think if we even keep run_etl, we want it to either only run its own class or to become a classmethod like get_subclass

Reply:
Definitely for a later PR, but I also agree.

I suspect we might end up ditching `run_etl` however.